routes/api/contacts.js

```diff
@@ -1,25 +1,39 @@
-const express = require('express')
+import express from "express";
+import Joi from "joi";
+import contactsService, { contactsSchema } from "../../models/contacts.js"
+import isValidId from "../../middlewars/isValidid.js";
+import isEmptyBody from "../../middlewars/isEmptyBody.js";
+import validateBody from "../../decorators/validateBody.js";
 
-const router = express.Router()
-
-router.get('/', async (req, res, next) => {
-  res.json({ message: 'template message' })
+const contactsAddSchema = Joi.object({
```

Comment:
Schema shouldn't be in routes file. It should be in separated file. You should validate email and phone properly